mcs/class/Makefile

```diff
@@ -39,6 +39,8 @@ build_SUBDIRS :=  \
 	System.Xml.Linq \
 	System.Runtime.InteropServices.RuntimeInformation \
 	System.IO.Compression \
+	System.Drawing			\
+	$(resgen_dir)			\
```

Comment:
I think this may have just been a fallout from the other change that broke parallel build, I'll try removing it.